website/meta/universe.json

```diff
@@ -4444,6 +4444,32 @@
             },
             "category": ["pipeline", "standalone", "scientific"],
             "tags": ["ner"]
+        },
+        {
+            "id": "hobbit-spacy",
+            "title": "Hobbit spaCy",
+            "slogan": "NLP for Middle Earth",
+            "description": "Hobbit spaCy is a custom spaCy pipeline designed specifically for working with Middle Earth and texts from the world of J.R.R. Tolkien.",
+            "github": "wjbmattingly/hobbit-spacy",
+            "pip": "en-hobbit",
+            "code_example": [
+                "import spacy",
+                "from spacy import displacy",
+                "",
+                "nlp = spacy.load('en_hobbit')",
+                "doc = nlp(text)"
```

Comment:
The code example is supposed to run as-is. Do you want to define `text` here to demonstrate that the pipeline works?

Reply:
And the `displacy` import could be removed as well, since you aren't visualizing anything here.

Reply:
Thanks! I have made those two corrections.